cypress/integration/regression/decision-tree-regression-pt2.spec.ts

```diff
@@ -0,0 +1,255 @@
+/* eslint-disable no-undef */
+import { datasetPage } from '../../page-objects/app/datasets-page'
+import { decisionTreesPage } from '../../page-objects/app/decision-trees-page'
+import { Timeouts } from '../../page-objects/lib/timeouts.cy'
+
+describe('Regression test of the decision tree', () => {
+  const includedVariants = '+5,041,176'
+  const datasetName = 'xl_PGP3140_wgs_NIST-4_2'
+  const filterName = '⏚Hearing Loss, v.5'
+  const decisionTreeName = 'new_decision_tree'
+
+  it('should expand all | step 18', () => {
```

Comment:
This test is too long and it's hard to understand what we're doing here. 
Let's join commands into the logic functions: test will become more readable and shorter.
As I can see we can also reuse the steps between other tests.